### ISSUE TITLE
Safeguards for accidental large cost runs

### ISSUE THREAD

**Issue by dpaleka** [pszemraj/lm-api#4]
Implement some sort of warning that you need to explicitly override to continue if the input file is unreasonably large.

This doesn't prevent failure modes when the script itself is called multiple times, but it's a good start.

**Comment by pszemraj** [comment]
good idea, can be also be an argparse switch that has no shorthand, something like `--safeguard-size` which defaults to 100 or so, then you have to reconfirm every `safeguard_size` number of queries.

I can take a look at this after my exams early Sept if you don't already have something/a better idea by then

**Comment by pszemraj** [comment]
did we resolve this with your PR or is there more work to be done?

**Comment by dpaleka** [comment]
I think the PR did not resolve that particular point, only some other issues with the OpenAI API
